release/release_tests.yaml

```diff
@@ -6812,6 +6812,7 @@
   stable: true
 
   env: gce
+  python: "3.8"
```

Comment:
python38 is the default, right?

Reply:
Oh, could be. I just noticed that the GCE `defaults.yaml` was previously pip installing the Ray p37 wheel, so I added this so that we can use the same python version for GCE, AWS and Azure.  Actually, I'm not totally sure what this `python` field does, let me look into it.